README.md

```diff
@@ -17,14 +17,14 @@
 
 Before installing and using microformats-ruby, you'll want to have Ruby 2.2.9 (or newer) installed. It's recommended that you use a Ruby version managment tool like [rbenv](https://github.com/rbenv/rbenv), [chruby](https://github.com/postmodern/chruby), or [rvm](https://github.com/rvm/rvm).
 
-microformats-ruby is developed using Ruby 2.5.0 and is additionally tested against versions 2.2.9, 2.3.6, and 2.4.3 using [Travis CI](https://travis-ci.org/indieweb/microformats-ruby).
+microformats-ruby is developed using Ruby 2.4.4 and is additionally tested against version 2.5.1 using [Travis CI](https://travis-ci.org/indieweb/microformats-ruby).
 
 ## Installation
 
 If you're using [Bundler](http://bundler.io) to manage gem dependencies, add microformats-ruby to your project's Gemfile:
 
 ```rb
-ruby '2.5.0'
+ruby '2.4.4'
```

Comment:
I've started removing this line from my `README.md` `Gemfile` examples since it might unwittingly imply that the stated version of Ruby is required (when in fact, the gem will work in other versions of Ruby).

It's a small thing and probably inconsequential, but figured I'd mention it.